pkg/cfn/builder/cluster_test.go

```diff
@@ -422,7 +463,7 @@ var _ = Describe("Cluster Template Builder", func() {
 		It("returns the template rendered as JSON", func() {
 			// the work actually gets done on the internal resource set
 			result, err := crs.RenderJSON()
-			Expect(err).NotTo(HaveOccurred())
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
```suggestion
			Expect(err).NotTo(HaveOccurred())
```

NotTo > ToNot

Reply:
actually idk anymore:

eksctl:
-> % rg NotTo | wc -l
362
-> % rg ToNot | wc -l
532

pctl:
-> % rg NotTo | wc -l
138
-> % rg ToNot | wc -l
73

we need to make a stand :joy:

Reply:
I followed whatever is in this file. I agree, I have no idea. :D

Reply:
following slack convo I'm going to change all the tests to `NotTo` on `main`. Can you change this file to `NotTo` as well please?

Reply:
Sure